docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,192 @@
+
+## InfluxDB plugin database field deprecation
+
+The `database` field in the provisioning file has been deprecated.
+This information will be stored in the `jsonData` field using the `dbName` property.
+The `database` field will be removed in the future to make InfluxDB consistent with other data sources.
+For more information and examples please refer to the [InfluxDB Provisioning docs]({{< relref "../datasources/influxdb/#provision-the-data-source" >}}).
+
+## Auth: Lock organization roles synced from auth providers
+
+_Generally available in all editions of Grafana._
+
+Grafana v9.4 provided the ability to configure synchronization of organization roles for each OAuth provider. With synchronization on, the organization role was applied to the user from the OAuth provider upon signing in. However, after the user signed in, you could still change the user’s organization role during the session.
+
+With this release, we are reinforcing organization role syncing behavior by introducing a new feature toggle called `onlyExternalOrgRoleSync`. Once enabled, users signing in to Grafana cannot change organization roles that have been synchronized from an external authentication provider, like Active Directory or Google OAuth. This can help ensure the right users maintain the right level of access at all times.
+
+This feature should be used if you want to enforce strict role synchronization from your auth provider to the organization roles.
+
+To use this feature, enable the `onlyExternalOrgRoleSync` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support. We'll also be automatically enabling this feature for Grafana Cloud instances over the upcoming weeks.
+
+You can also _prevent_ the synchronization of organization roles from a given authentication provider. Learn more in our [skip org role sync]({{< relref "../setup-grafana/configure-grafana/#authgrafana_com-skip_org_role_sync/" >}}) documentation.
+
+## Reporting UI adapted to match the new navigation style
+
+_Generally available in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+We updated the reporting UI to better fit the new navigation style, adding a horizontal slider and moving the **Preview** and **Send** buttons to the Action section in the page. We also fixed the alignment of the different sections.
+
+{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="Grafana new report UI" >}}
+
+## Experimental support for using JWTs as auth method
+
+_Experimental in Grafana Open Source and Enterprise._
+
+This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file.
```

Comment:
```suggestion
This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file.
```